features/bootstrap/VirtualMetricHandleContext.php

```diff
@@ -22,9 +22,11 @@ public function iAddAVirtualMetric()
         $this->page = new MetricsConfigurationListingPage($this);
         $this->assertFind('css', 'a[class="btc bt_success"]')->click();
         $this->page = new MetricsConfigurationPage($this);
+        sleep(60);
```

Comment:
is this sleep intended ?

Reply:
`spin` method must be used

Reply:
Even tho i used it, it seems like the tests don't pass for some reasons.

Reply:
ah no it works now, for 70 seconds